src/main/java/gift/option/Option.java

```diff
@@ -59,6 +60,14 @@ public void update(String name, int quantity) {
         this.quantity = quantity;
     }
 
+    public void subtract(int quantity) {
+        if (quantity < 0) {
+            throw new IllegalArgumentException(OPTION_SUBTRACT_NOT_ALLOWED_NEGATIVE_NUMBER);
+        }
+
+        update(name, this.quantity - quantity);
```

Comment:
update 메서드를 재사용하지 않아도 될 것 같습니다.
```suggestion
        this.quantity - quantity;
```